hyperspy/io_plugins/bcf.py

```diff
@@ -678,6 +741,17 @@ def get_spectra_metadata(self, index=0):
         index -- index of hypermap/spectra (default 0)
         """
         return self.spectra_data[index]
+    
+    def calc_real_time(self):
+        """calculate and return real time for whole hypermap
+        in seconds
+        """
+        line_cnt_sum = np.sum(self.line_counter)
+        line_avg = self.image.dsp_metadata['LineAverage']
+        pix_avg = self.image.dsp_metadata['PixelAverage']
+        pix_time = self.image.dsp_metadata['PixelTime']
+        width = self.image.width
+        return line_cnt_sum * line_avg * pix_avg * pix_time * width / 1000000.0
```

Comment:
The return value is `numpy.float64`, changing it to `float` should avoid displaying too many (and irrelevant) digits in the metadata tree (that happen with one of my files).